### DISCUSSION TITLE
Saving Prefab to immutable folder is not allowed 가 유니티에서 시작 후 저장할때 마다 뜨고 있습니다.

### DISCUSSION THREAD

**Discussion by gwililgwi** [naverz/zepeto-studio-kor#572] [5. Q&A]
Saving Prefab to immutable folder is not allowed: Packages/zepeto.character/resources/zepeto/ZepetoBaseModel.prefab
UnityEditor.EditorApplication:Internal_CallGlobalEventHandler ()

위 코드가 유니티에서 플레이를 눌러 테스트 한 뒤 저장을 누르면 계속해서 오류 로그가 뜨고 있습니다.
문제가 되는 것은 없지만, 추후 심사에 영향을 주게될 것 같아 남겨요!!

____

월드 패키지 버전: 1.1.5
제페토 앱: 최신
개발환경: Window 10

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 상세한 제보 감사합니다.
Known Issue로 확인 및 수정 작업 진행 중입니다.
빌드는 가능하며 해당 오류로 인해 심사에는 영향 없는점 참고 부탁드립니다.

**Reply by Zepeto-TechEvangelist** [reply]
월드 1.6.0 버전에서 해당 이슈 수정 완료되었습니다 :)
